src/components/filopplaster/filopplasterModal/FilopplasterModal.tsx

```diff
@@ -13,23 +25,19 @@ import env from '../../../utils/environment';
 import { logger } from '../../../utils/logger';
 import { post } from '../../../data/fetcher/fetcher';
 import Datovelger from '../../kvittering/datovelger/Datovelger';
-import { useAppStore } from '../../../data/stores/app-store';
-import { generateId } from '../../../utils/random';
-import TransportmiddelKvittering from '../../kvittering/TransportmiddelKvittering';
 
 const FilopplasterModal: React.FC = () => {
   Modal.setAppElement('#root'); // accessibility measure: https://reactcommunity.org/react-modal/accessibility/
 
   const [laster, settLaster] = useState<boolean>(false);
   const [dato, settDato] = useState<Date | null>(null);
   const [beløp, settBeløp] = useState<number | null>(null);
```

Comment:
Beløp bør være en string. Slik det er nå brukes input for beløp som en uncontrolled komponent, og dette er anti-pattern i React: https://reactjs.org/docs/uncontrolled-components.html

Reply:
+1